Misc/NEWS.d/next/Library/2025-11-18-14-39-31.gh-issue-141570.q3n984.rst

```diff
@@ -0,0 +1,3 @@
+In ``_colorize.can_colorize``, support file objects rising
+:exc:`OSError` from the :meth:`~io.IOBase.fileno` method, as docuemtned. An
+example of such an object is a ``mod_wsgi`` logger.
```

Comment:
```suggestion
Support :term:`file-like object` raising :exc:`OSError` from :meth:`~io.IOBase.fileno` in color 
detection (``_colorize.can_colorize``). This can occur when `sys.stdout` is redirected.
```